scss/styles.scss

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
i would write `*  {` instead of body, because we are going to use this font in our entire app. (Head, body, footer...)